ci/clean-test-output.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# This script remove test and benchmark output and displays disk usage.
```

Comment:
Minor thing but should this live in the workflows directory?  Thats where I tend to put stuff like this which also helps keep the root clean

Reply:
Sounds good to me. Do you want me moving them all in this PR?

Reply:
BTW rust-lang/rust puts them under `src/ci`.

Reply:
BTW rust-lang/rust puts them under `src/ci`. I am okay with all these options.